eventhubs/data-plane/azure-eventhubs-eph/src/main/java/com/microsoft/azure/eventprocessorhost/InMemoryLeaseManager.java

```diff
@@ -250,9 +253,10 @@ public void notifyOnSteal(String expectedOwner, String partitionId, Callable<?>
                 leaseToRelease.setOwner("");
                 leaseInStore.setExpirationTime(0);
                 leaseToRelease.setExpirationTime(0);
-            } else {
-                // Lease was lost, intent achieved.
             }
+//            else {
+//                // Lease was lost, intent achieved.
+//            }
```

Comment:
This makes me uneasy - there is a chance that you've restructured the conditional ordering by doing this (I'd need to review a bit more to be certain). Maybe @conniey can double-check here.